packages/ai-rules/.ruler/frontend/api-patterns.md

```
@@ -0,0 +1,427 @@
+## Technology Stack
+
+- **React Query** (@tanstack/react-query) for data fetching and caching
+- **Zod** for runtime schema validation
+- **parsedApi.ts** for type-safe HTTP client
+
+## parsedApi.ts Methods
+
+```typescript
+import { get, post, put, patch, remove } from "@src/appV2/api/parsedApi";
+
+// All methods support:
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)